src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -408,7 +414,8 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
             //If data type is specified
             if (strArr.size() == 4) {
                 attrDataType = strArr[3];
-                if (attrDataType != "int8" && attrDataType != "int16" && attrDataType != "int32" && attrDataType != "float") {
+                if (attrDataType != "int8" && attrDataType != "int16" && attrDataType != "int32" &&
+                    attrDataType != "float") {
                     frontend_logger.log("Data type not recognized", "error");
```

Comment:
Shall we also add `attrDataType` to the log message here?

Reply:
This section was there originally. the only change that has come from my PR is that adding a vertical space